src/graphic/gl/initialize.cc

```diff
@@ -133,7 +133,11 @@ SDL_GLContext initialize(
 	// LeakSanitizer reports a memory leak which is triggered somewhere above this line, probably
 	// coming from the gaphics drivers
 
-	if (err != GLEW_OK) {
+	// err == GLEW_ERROR_NO_GLX_DISPLAY is a workaround for crash on wayland
+	// https://github.com/nigels-com/glew/issues/172
+	if (err == GLEW_ERROR_NO_GLX_DISPLAY) {
```

Comment:
SDL is already initialised here, so we can check whether we're actually using wayland: (not tested)

```suggestion
	if (err == GLEW_ERROR_NO_GLX_DISPLAY && std::strcmp(SDL_GetCurrentVideoDriver(), "wayland") == 0) {
```